defender/gasless-nft-minting/README.md

```diff
@@ -0,0 +1,168 @@
+- Click on the existing Relay that you would like to use for deployments
+- Click on the settings gear/cog **⚙** at the top middle and choose **+ Create new API key**
+- Copy your Ethereum Address, API key, and Secret key to a local file (you will **NOT** be able to view your Secret key again after this message box goes away)
+- Make sure that you really did copy your API key and Secret key to a local file
+- Check the box for **I’ve written down the secret key** and select **Close**
+
+### Add Funds to the Relay
+
+- In your [Defender account](https://defender.openzeppelin.com/)
+- Click on the **Relay** link on the left side
+- Find the Relay that deployed the NFT contract, click on the **copy address** button
+- Transfer funds to that address to cover the deployment and NFT minting costs
+
+### Local Code Setup
+
+In the `gasless-nft-minting` directory, perform the following steps:
+
+- A secrets file can be created for each stage of production. We will create one for development:
+  - Copy and rename the `sample.secrets.yml` to `.secrets/dev.yml`
+  - Modify the four lines in the `.secrets/dev.yml` file, replacing the portion in the angle brackets `<>` with your Defender API keys and Secret keys, as indicated:
+  - `defender-api-key: <API Key goes here>`
+  - `defender-api-secret: <Secret key goes here>`
+  - `relay-api-key: <Relay API Key goes here>`
+  - `relay-api-secret: <Relay Secret key goes here>`
+- Change directories to the Contract Wizard Deployer
+  - `cd gasless-nft-minter`
+- Run `yarn install` to install the necessary Node packages
+
+### Solidity Files
+
+The `DemoNFT` contract in the `gasless-nft-minting/contracts` directory is a custom contract that was made with OpenZeppelin's [Contracts Wizard](https://wizard.openzeppelin.com/). Modifications were made to make it [ERC2771-compatible](https://docs.openzeppelin.com/contracts/4.x/api/metatx#ERC2771Context) (support gasless / meta transactions).
+
+### Deploying Contracts
+
+Contracts will be deployed using the Defender Relay that you configured earlier.
+
+- The `MinimalForwarder` contract must be deployed first
+  - `yarn deploy contract --contract-name MinimalForwarder`
+- Take the address of the `MinimalForwarder` and provide it as a constructor for the NFT contract
+  - `yarn deploy contract --contract-name DemoNFT "0xMINIMAL_FORWARDER_ADDRESS"`
+- Make a note of both newly deployed contract addresses. They will need to be added to the config file mentioned below
+
+### Serverless Deployment Setup :rocket:
+
+Now that the contracts are deployed, they can be added to the config file so that the Autotasks know which contracts they are signing for and forwarding to. In the `gasless-nft-minting` directory, perform the following steps:
+
+- A config file can be created for each stage of production. We will create one for development
+  - Copy and rename `sample.config.yml` to `config.dev.yml`
```

Comment:
```suggestion
  - Copy and rename `example.config.yml` to `config.dev.yml`
```

Reply:
renamed the file. 9daa9144369044130f497f8c4a41219aab381ec9